Div.java

```diff
@@ -0,0 +1,9 @@
+package com.edocti.jintro.lab01;
```

Comment:
From a styling perspective this is perfect 👍 

What you could do (optional) is to check whether `args.length >= 2`. Otherwise you could display an error message. `NumberFormatException` is possible, but this is not that important, since we didn't yet learn about exceptions